plugins/extractors/postgres/postgres_test.go

```diff
@@ -174,7 +171,7 @@ func getExpected() []models.Record {
 					"grants": []interface{}{
 						map[string]interface{}{
 							"user":            "test_user",
-							"privilege_types": "INSERT,SELECT,UPDATE,DELETE,TRUNCATE,REFERENCES,TRIGGER",
+							"privilege_types": []interface{}{"INSERT", "SELECT", "UPDATE", "DELETE", "TRUNCATE", "REFERENCES", "TRIGGER"},
```

Comment:
@StewartJingga thought of this to support an array of privileges a user has. wdyt???